Automattic/sensei
# Reload page after reset progress call

Fixes #2860

#### Changes proposed in this Pull Request:

* Reload page on Learner Management after clicking Reset progress, to see the updated Status and Date for the learner.

#### Testing instructions:

* Go to Learner Management
* Click `Reset progress` for a learner.
* Page should reload
